playground/app.jsx

```diff
@@ -224,31 +224,33 @@ const App = ({location: {pathname, query}}) => (
 
         <div className="row">
           <div className="col-md-8">
-            <Autocomplete
-              autoFocus
-              data={{
-                items: [
-                  {item: 'meow', itemInfo: 'hard'},
-                  {item: 'meowing', itemInfo: 'harder'}
-                ]
-              }}
-              scrollTo
-              id="test"
-              value="purrr"
-              query={query}
-              onSelectSuggestion={suggestion => { console.log('You have chosen ', suggestion); }} // eslint-disable-line no-console
-              placeholder="Type something..."
-              suggestionsFooter={<Link href="hi">No suggestions found?</Link>}
-              label="Autocomplete"
-              isRequired
-              requiredLabel="Required"
-              name="autocomplete" />
+            <form>
+              <Autocomplete
+                autoFocus
+                data={{
+                  items: [
+                    {item: 'meow', itemInfo: 'hard'},
+                    {item: 'meowing', itemInfo: 'harder'}
+                  ]
+                }}
+                scrollTo
+                id="test"
+                value="purrr"
+                query={query}
+                placeholder="Type something..."
+                suggestionsFooter={<Link href="hi">No suggestions found?</Link>}
+                label="Autocomplete"
+                isRequired
+                requiredLabel="Required"
+                name="autocomplete" />
+            </form>
           </div>
         </div>
 
         <div className="row">
           <div className="col-md-8">
             <Autocomplete
+              onKeyDown={e => { console.log('test', e); }}
```

Comment:
We get a eslint warning here, maybe disable it for this line

Reply:
I decided to add the app.jsx to the eslintignore since it is very common to test component functions using console log.

Reply:
Makes sense, however I see the error, when I start the package.

Reply:
Try pulling, I commited this after you started your review.